osu.Desktop/DiscordRichPresence.cs

```diff
@@ -139,9 +186,38 @@ private void updateStatus()
             client.SetPresence(presence);
         }
 
+        private void onJoin(object sender, JoinMessage args)
+        {
+            game.Window?.Raise();
+            Logger.Log($"Received room secret from Discord RPC Client: {args.Secret}", LoggingTarget.Network, LogLevel.Debug);
+
+            // Stable and Lazer share the same Discord client ID, meaning they can accept join requests from each other.
+            // Since they aren't compatible in multi, see if stable's format is being used and log to avoid confusion.
+            // https://discord.com/channels/188630481301012481/188630652340404224/1214697229063946291
+            if (args.Secret[0] != '{')
+            {
+                Logger.Log("osu!stable rooms are not compatible with lazer.", LoggingTarget.Network, LogLevel.Important);
```

Comment:
Don't quite know if the direct reference to "lazer" here is good, but I think this still conveys the message. Suggestions are appreciated.

Reply:
This assume anything that doesn't start with `{` is a room from osu!stable, but if it starts with `{` then it says "could not join multiplayer room". Might as well just group them and say invitation is invalid / incompatible.